kube-derive/src/custom_resource.rs

```diff
@@ -445,12 +454,13 @@ fn process_status(root_ident: &Ident, status: &Option<String>, visibility: &Visi
                 }
             }
         };
-        (status_field, status_default, status_impl)
+        StatusInformation {
+            field,
+            default,
+            impl_hasstatus,
+        }
     } else {
-        let status_field = quote! {};
-        let status_default = quote! {};
-        let status_impl = quote! {};
-        (status_field, status_default, status_impl)
+        StatusInformation::default()
```

Comment:
Nit: I'd prefer specifying these to be explicitly empty

Reply:
Okay, sure. Are you fine with `TokenStream::new()` or would you like to keep `quote! {}`?

As this runs during compile phase I'd like to keep it as short as possible.

Reply:
Either works for me.

Reply:
empty `quote! {}` fits whats around i guess

Reply:
I've changed it. Let me know if that works for you.